telescope/config.yaml

```diff
@@ -16,7 +16,7 @@ docker:
   airflow_report: &airflow_report
     - /bin/sh
     - -c
-    - curl -ssl https://raw.githubusercontent.com/astronomer/telescope/dev/airflow_report.pyz > airflow_report.pyz && chmod +x airflow_report.pyz && PYTHONWARNINGS=ignore ./airflow_report.pyz
+    - curl -ssl https://raw.githubusercontent.com/astronomer/telescope/add_user_report/airflow_report.pyz > airflow_report.pyz && chmod +x airflow_report.pyz && PYTHONWARNINGS=ignore ./airflow_report.pyz
```

Comment:
I'll revert this to "main", but also probably switch it to packaging once I've figured that out 🤔